app/presenters/bulk_exports_presenter.rb

```diff
@@ -16,7 +16,7 @@ def initialize
   private
 
   def init_exported_at_message
-    export_executed_at = DefraRuby::Exporters::BulkExportFile.first&.created_at
+    export_executed_at = DefraRuby::Exporters::BulkExportFile.first&.created_at&.strftime("%-d %B %Y")
```

Comment:
I know you didn't request a review from me but here I am anyway! The engine actually has a time formats initializer with some date presets, so you can do `.to_formatted_s(:day_month_year)` rather than specifying new ones. That's what we've done in the exemptions list on the registration details page.

Reply:
Fixed in c67bf02.